site/installing.md

```diff
@@ -204,7 +206,7 @@ Where,
 
 * `SCOPE_SERVICE_TOKEN=<token>` is the token you obtained after you signed up for Weave Cloud.
 
-Note that you will need to launch Scope onto every node that you want to monitor.
+Note that you will need to launch Scope on every node that you want to monitor.
 
```

Comment:
you can use the new note format here: 

`> note:`  and it will add a shaded box around this. 

I

Reply:
I'll leave the formatting changes to somebody more qualified, i.e. you :)

Reply:
Ok I'll fix that up later.  We need to add V3 here as well...